src/types.d.ts

```diff
@@ -50,16 +50,19 @@ export interface EnterpriseCustomer {
 }
 
 export interface EnterpriseFeatures {
-  enterpriseLearnerBffEnabled: boolean;
+  enterpriseLearnerBffEnabled?: boolean;
   [key: string]: boolean;
 }
 
 export interface EnterpriseLearnerData {
-  enterpriseCustomer: Types.EnterpriseCustomer;
-  activeEnterpriseCustomer: Types.EnterpriseCustomer;
-  allLinkedEnterpriseCustomerUsers: any[];
-  staffEnterpriseCustomer: Types.EnterpriseCustomer;
+  enterpriseCustomer: Types.EnterpriseCustomer | null;
+  activeEnterpriseCustomer: Types.EnterpriseCustomer | null;
+  activeEnterpriseCustomerUserRoleAssignments: any[];
```

Comment:
[clarification/consideration] Keep in mind that properties now required in `EnterpriseLearnerData` must be returned/derived from the BFF-enabled cases, as well. Does `activeEnterpriseCustomer`, `activeEnterpriseCustomerUserRoleAssignments`, and `enterpriseCustomerUserRoleAssignments` need to be included in the BFF API responses via https://github.com/openedx/enterprise-access/pull/624?

Or is it expected/OK that, in the BFF-enabled cases, these data are missing? If expected, should the additional fields be made optional?